packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -234,19 +234,37 @@ describe('Collection aggregations tab', function () {
       const text = await textElement.getText();
       return text === '(Sample of 100 documents)';
     });
+  });
 
-    // save as a view
-    // TODO: This is currently broken, so will have to test at a later stage
-    /*
-    //#save-pipeline-actions
-    //a=Create a view'
-    '[trackingid="create_view_modal"]'
-    '#create-view-name'
-    '[trackingid="create_view_modal"] [role=dialog] > div:nth-child(2) button:first-child'
-    */
+  it('supports saving pipeline as a view', async function () {
+    await browser.focusStageOperator(0);
+    await browser.selectStageOperator(0, '$match');
+    await browser.setAceValue(Selectors.stageEditor(0), `{ i: 5 }`);
+
+    await browser.clickVisible(Selectors.SavePipelineActions);
+
+    await browser.clickVisible(Selectors.SavePipelineActionsCreateView);
 
-    // browse to the view
-    // TODO
+    await browser.waitForAnimations(Selectors.CreateViewNameInput);
+    const viewNameInput = await browser.$(Selectors.CreateViewNameInput);
+    await viewNameInput.setValue('my-view-from-pipeline');
+
+    const createButton = await browser
+      .$(Selectors.CreateViewModal)
+      .$('button=Create');
+
+    await createButton.click();
```

Comment:
There's a small chance it won't be enabled yet, I suppose. Although I think the other modals would just clickVisible on a selector and I don't think that waits for enabled explicitly anyway 🤔

Reply:
I guess it boils down to whether react would respond synchronously to the onchange in the text input or if there will be a chance for other code (such as this) to execute in between.

Reply:
A valid point, I'm thinking I'll keep it in line with other cases for now then if that's fine